sdk/core/azure-core/azure/core/pipeline/policies/_authentication.py

```diff
@@ -95,11 +95,11 @@ def on_request(self, request):
 
 
 class AzureKeyCredentialPolicy(SansIOHTTPPolicy):
-    """Adds an API key header for signing requests.
+    """Adds a key header for signing requests.
 
     :param credential: The credential used to authenticate requests.
     :type credential: ~azure.core.credentials.AzureKeyCredential
-    :param str key_header: The name of the API key header used for signing requests.
```

Comment:
API keys can be sent in headers or in query strings (today, we only have services that use headers). Is there a name we can use for the parameter that can reflect this/not break if/when we have to support query string?

Reply:
I've been thinking of a name for a while and have not come up with anything here. @maririos  do you have any thoughts? The best I can come up with is key_param_name, which is awful

Reply:
Sometimes a shorter parameter name can help. What about simply calling it `name`? And if we end up having to distinguish between headers and query parameters, when can add a `location` parameter and default it to 'header'.

Reply:
`name` works for me